src/modules/password/resetService.js

```diff
@@ -0,0 +1,84 @@
+const { INVALID_SMS_CODE_MESSAGE, INVALID_PHONE_NUMBER } = require('./constants');
```

Comment:
`resetService` is very bad naming of the file
And on top of that - you created separate sms module
So please refactor your modules (sms logic should be inside appropriate module service)